insights/parsers/ssl_certificate.py

```diff
@@ -251,3 +253,25 @@ class NginxSSLCertExpireDate(CertificateInfo):
         '/a/b/c.pem'
     """
     pass
+
+
+@parser(Specs.mssql_tls_cert_enddate)
+class MssqlTLSCertExpireDate(CertificateInfo):
+    """
+    .. note::
+        Please refer to its super-class :class:`insights.parsers.ssl_certificate.CertificateInfo` for more
+        details.
+
+    It parses the output of ``openssl x509 -in mssql_tls_cert_file -enddate -noout``.
+
+    Sample output of ``openssl x509 -in mssql_tls_cert_file -enddate -noout``::
+
+        notAfter=Dec 4 07:04:05 2035 GMT
+
+    Examples:
+        >>> type(mssql_date_info)
+        <class 'insights.parsers.ssl_certificate.MssqlTLSCertExpireDate'>
+        >>> mssql_date_info['notAfter'].datetime
```

Comment:
@wushiqinlou As we are specifically looking for the value of 'notAfter' here can we include that check in the parser logic instead of checking `'notAfter' in date_info` in the rule code ? The basics idea is to return `None` is 'notAfter' does not exist.

Reply:
@psachin - the superclass is a dict, a `KeyError` will be thrown when fetching a non-existed key. I think this is okay for a `dict` parser (but a bit weird to enforce the obj[key] returns None by default), of cause, `.get()`  would return `None` in that case as well.

Reply:
@xiangce Yes, I wanted to suggest `.get()` what do you think ?